physical/raft/fsm.go

```diff
@@ -678,24 +677,24 @@ func (f *FSM) ApplyBatch(logs []*raft.Log) []interface{} {
 	// Do the unmarshalling first so we don't hold locks
 	var latestConfiguration *ConfigurationValue
 	commands := make([]interface{}, 0, numLogs)
-	for _, l := range logs {
```

Comment:
Nit: I have a very mild preference that variable names not shadow package names, just for the sake of readability, which is why I changed all these `log` references in the original PR. Not a big deal by any means.